pkg/plugins/resources/githubrelease/main.go

```diff
@@ -50,13 +54,18 @@ func New(spec interface{}) (*GitHubRelease, error) {
 		return &GitHubRelease{}, err
 	}
 
+	if !newSpec.OriginalVersion {
+		logrus.Warningf("%s\n\n", DeprecatedSemverVersionVersion)
```

Comment:
```suggestion
		logrus.Warningf("%s\n\n", DeprecatedSemverVersionMessage)
```
(same elsewhere)

Reply:
This one has been solve another commit. thank you for catching this